artiq/dashboard/moninj.py

```diff
@@ -614,6 +615,7 @@ def build(self):
             def run(self):
                 self.core.reset()
                 cfg = self.core_cache.get("_{cpld}_cfg")
+                delay(40*ms)
```

Comment:
Seem arbitrary.
Why do you need to actually `core.reset()`? And why don't you move that (or better just `break_realtime()`) to just above the `cpld.init()`?